### ISSUE TITLE
pgwatch2_replication_in_recovery_int is always set to 0

### ISSUE THREAD

**Issue by DreamyProtect** [cybertec-postgresql/pgwatch2#475]
Hi !

I'm trying to set up some monitoring on a basic semi-HA setup of postgresql nodes using pgwatch2 with the prometheus metrics exposition feature however there seems to be a problem where the metric `pgwatch2_replication_in_recovery_int` is always set to 0 and I can't grasp my hands on why.

The postgresql setup is pretty basic: 1 master node with a VIP set up using keepalived and a read-only hotstandby replicating the primary. When the primary is down, the VIP will go on the secondary that will stay in hotstandby and only accept read-only connections. This is not pure HA but this will do for now for our needs.

We've set up pgwatch2 as a systemd service on both nodes in prometheus mode, created the pg_stat_statement and plpython3u extensions and rolled out the following helpers after installing PL/Python :
- "get_stat_activity/9.2/metric.sql"
- "get_stat_replication/9.2/metric.sql"
- "get_wal_size/10/metric.sql"
- "get_sequences/10/metric.sql"
- "get_load_average/9.1/metric.sql"

So the pgwatch2 is not running as a superuser, it also is member of the role `pg_monitor`.

When querying the databases directly we get the following results for the pg_is_in_recovery() function:

- On the primary:
```
postgres=# SELECT * from pg_is_in_recovery();
 pg_is_in_recovery
-------------------
 f
(1 row)
```

- On the secondary:
```
postgres=# SELECT * from pg_is_in_recovery();
 pg_is_in_recovery
-------------------
 t
(1 row)
```

So the results seems good to me from a database perspective. I've also checked the logs in postgresql and pgwatch2 but nothing stands out.

I tried digging a little bit into the code of pgwatch2 in order to understand what's happenning and as I understood it, the following script is responsible for fetching that metric: https://github.com/cybertec-postgresql/pgwatch2/blob/master/pgwatch2/metrics/db_stats/14/metric.sql but the code seems fine to me.

Do you have any idea on what's causing the problem ?

Thanks in advance for your help !

** Additionnal informations **
Both postgresql are running on Debian 11 (bullseye) in version 14.2.
pgwatch2 is running in version 1.8.5

Here's my instances.yaml:
```yaml
---
#
# Ansible managed
#

- custom_tags:
    env: preproduction
  dbname: ''
  dbname_include_pattern: .
  dbtype: postgres-continuous-discovery
  host: localhost
  is_enabled: true
  is_superuser: false
  password: XXX
  port: 5432
  preset_metrics: prometheus
  sslmode: disable
  unique_name: preproduction
  user: pgwatch2
```

**Comment by pashagolub** [comment]
Hello. 

Thanks for your request. I cannot find any mention of `pgwatch2_replication_in_recovery_int` niether in the sources nor the documentation. Would you please recheck the name and/or define the metric?

Thanks in advance

**Comment by DreamyProtect** [comment]
I think the metric should be `in_recovery_int` and I guessed that `pgwatch2_replication_` was added by the prometheus mode: https://github.com/cybertec-postgresql/pgwatch2/search?q=in_recovery_int.

Actually I double checked just now and it seems like my Prometheus scrapping is the cause I only saw it being valued to 0, because when I curl my primary it is set to 0 and when I curl the secondary it is well set to 1.

My bad sorry ! I'm closing this since there's no real issue.
